### PR TITLE
Simplified the process of getting the user's manager details and fixed a potential error

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23733

## Description
- Improved the process of getting the user's manager details. This may also improve the success rate of retrieving the manager details.
- Fixed a potential scenario where we would attempt to disable the user when Active Directory is not enabled.

## Screenshots
image

## Minimum version of Demisto
5.0.0

## Does it break backward compatibility?
No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml [hunk 1]
```diff
@@ -1,3365 +1,3046 @@
...
+  description: Endpoint OS
+  type: string
+- contextPath: Endpoint.IP
+  description: List of endpoint IP addresses
+- contextPath: Endpoint.MAC
+  description: List of endpoint MAC addresses
+- contextPath: Endpoint.Domain
+  description: Endpoint domain name
+  type: string
+sourceplaybookid: Impossible Traveler
```
Comment (Itay4) [inline Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:3043]: did you not run format on this playbook?
remove that field

Comment (idovandijk) [reply Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:3043]: I ran format (hence the huge change all over the playbook), but apparently it doesn't remove it, so I'll remove manually

**Comment by kirbles19** [comment]
@idovandijk Done.

File: Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml [hunk 2]
```diff
@@ -1,3365 +1,3045 @@
...
+      name: Should the source IPs be blocked automatically?
+      description: Checks whether the source IPs that the user used to login from
+        can be blocked automatically, according to the playbook inputs.
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '79'
+      yes:
```
Comment (yaakovi) [inline Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:427]:
```suggestion
      'yes':
```

File: Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml [hunk 3]
```diff
@@ -1,3365 +1,3045 @@
...
+      - '71'
+    separatecontext: false
+    conditions:
+    - label: yes
```
Comment (yaakovi) [inline Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:431]:
```suggestion
    - label: 'yes'

```

File: Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml [hunk 4]
```diff
@@ -1,3365 +1,3045 @@
...
+      name: Are there coordinates for the source IPs?
+      description: Checks whether coordinates were retrieved for the IPs that the
+        user used to login.
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '54'
+      yes:
```
Comment (yaakovi) [inline Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:543]:
```suggestion
      'yes':

```

File: Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml [hunk 5]
```diff
@@ -1,3365 +1,3045 @@
...
+      - '58'
+      - '59'
+    separatecontext: false
+    conditions:
+    - label: yes
```
Comment (yaakovi) [inline Packs/ImpossibleTraveler/Playbooks/playbook-Impossible_Traveller.yml:548]:
```suggestion
    - label: 'yes'

```

**Pull request merged**